lib/FSi/DoctrineExtensions/PropertyManipulator.php

```diff
@@ -27,21 +29,19 @@ class PropertyManipulator
 
     /**
      * @param object $object
-     * @param string $property
```

Comment:
we shouldn't split contents of any docblock like this

Reply:
Well I have only left the `@param object $object` until we can use the `object`  typehint. I can remove other fields and leave just this one.